run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.tagCO

```diff
@@ -116,7 +116,7 @@ Mask fractions:              false
     --> CMIP6_AIRCRAFT         :       false    # 1850-2100
     --> CMIP6_SHIP             :       false    # 1850-2100
 # ----- BIOMASS BURNING EMISSIONS -----------------------------
-    --> QFED2                  :       true     # 2000-2020
+    --> QFED2                  :       false    # 2000-2020
```

Comment:
@msulprizio: TagCO uses QFED2 because the GFED extension biomass emissions cannot be masked.  But this is OK to disable QFED2 since we are deprecating the tagCO simulation.